app/statistics/server/lib/statistics.js

```diff
@@ -18,13 +18,23 @@ import {
 	Sessions,
 	Integrations,
 	Uploads,
+	LivechatDepartment,
+	EmailInbox,
+	LivechatBusinessHours,
 } from '../../../models/server/raw';
 import { readSecondaryPreferred } from '../../../../server/database/readSecondaryPreferred';
 import { getAppsStatistics } from './getAppsStatistics';
 import { getServicesStatistics } from './getServicesStatistics';
 import { getStatistics as getEnterpriseStatistics } from '../../../../ee/app/license/server';
 import { Team, Analytics } from '../../../../server/sdk';
 import { getSettingsStatistics } from '../../../../server/lib/statistics/getSettingsStatistics';
+import {
+	CannedResponseRaw,
+	LivechatPriorityRaw,
+	LivechatTagRaw,
+	LivechatUnitMonitorsRaw,
+	LivechatUnitRaw,
+} from '../../../../ee/app/models/server';
```

Comment:
yep.. ideally CE code should not depend on EE code.. 

a possible solution to this would be creating an "EnterpriseStatitics" service, so calling it to get the values, if it is not there, a `null` would be returned..

Reply:
Yep, like the idea 👀

@cauefcr , i can help you there, let's find a time for doing it 🥳